boards/arm64/fvp_baser_aemv8r/fvp_baser_aemv8r.yaml

```diff
@@ -5,8 +5,11 @@ identifier: fvp_baser_aemv8r
 name: FVP Emulation FVP_BaseR_AEMv8R
 arch: arm64
 type: sim
+simulation: armfvp
```

Comment:
Is that our fixes should be merged right before this PR? Or I am afraid this would block others?

And I think we need to add a warning or alerting when users run `twister `locally  without installing any FVP... I'll take a look

Reply:
> Is that our fixes should be merged right before this PR? Or I am afraid this would block others?

Yes, the fixes need to be merged first -- https://github.com/zephyrproject-rtos/zephyr/issues/43852#issuecomment-1108231033.

> And I think we need to add a warning or alerting when users run twister locally without installing any FVP.

Indeed, we should.

Reply:
> And I think we need to add a warning or alerting when users run twister locally without installing any FVP.

Added https://github.com/zephyrproject-rtos/zephyr/pull/45099/commits/8683cac21d2fbdd73d721c8f9ee54dbfdcbb0e9b, which makes twister skip the platforms that require the ARM FVP when the `ARMFVP_BIN_PATH` environment variable is not set.